src/blocks/calendar.scss

```diff
@@ -0,0 +1,60 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: #000;
+}
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  height: $calendar-height;
+  @for $day-number from 1 to $day-count {
+    &--month-length_#{$day-number} {
+      .calendar__day:nth-child(n + #{$day-number + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  @each $day, $index in $week-days {
+    &--start-day_#{$day} {
+      .calendar__day:nth-child(1) {
+        margin-left: $start-shift * $index;
+      }
+    }
+  }
+
+  &__mockup {
+    display: flex;
+    justify-content: flex-start;
+    align-content: center;
+    flex-wrap: wrap;
+    padding: $mockup-padding;
+    gap: $gap-size;
+    min-width: $mockup-size;
+    max-width: $mockup-size;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square($day-size);
```

Comment:
Do you use it somewhere else?

Reply:
No , it was used just ones  - for purpose of practice  in sass :))

Reply:
regarding $calendar-height: 100vh -  I was excited by possbilities of sass and tried to implemet as many variables, extends , mixins as possible . For sure - that is should be  direct setting of the field.